File: Programmers/Lv0/배열회전시키기.py
def solution(numbers, direction):
    if direction == "right":
        numbers.insert(0, numbers[len(numbers) - 1])
        numbers.pop()
    else:
        numbers.append(numbers[0])
        numbers.pop(0)
    return numbers

# list슬라이싱
def solution(numbers, direction):
    return [numbers[-1]] + numbers[:-1] if direction == 'right' else numbers[1:] + [numbers[0]]

File: Programmers/Lv0/배열회전시키기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> numbers, string direction) {
    if (direction == "right")
    {
        numbers.insert(numbers.begin(), numbers.back());
        numbers.pop_back();
    }
    else if (direction == "left")
    {
        numbers.push_back(numbers.front());
        numbers.erase(numbers.begin());
    }
    return numbers;
}